job-builder/jobs/dependencies.yaml

```diff
@@ -0,0 +1,68 @@
+---
+- job-template:
+    name: '{configuration}-nightly-x86_64'
```

Comment:
Could you try to find a way to uses variables to avoid hard-coding architectures? Ideally, we'd be able to use this set of files for all architectures Kata runs on.